src/script/components/app-card.ts

```diff
@@ -208,6 +209,36 @@ export class AppCard extends LitElement {
       }
     `;
 
+    const contentCardcss = css`
+      :host {
+        background: white;
+        display: flex;
+        color: var(--primary-color);
+        justify-content: center;
+      }
+
+      fast-card.content-card {
+        background-color: white;
+        padding: 1rem 0;
+        width: min(100%, 1366px);
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        box-shadow: none;
+        border-radius: 0;
+        border-bottom: 0.67681px solid #e5e5e5;
+      }
+
+      .content-card__header {
```

Comment:
The double underscore is part of BEM naming convention, but we've decided against using that convention. I'd rather see `.content-card-header` or `.content-card .header` here.

Reply:
Fixed it